### ISSUE TITLE
Feature Request - Wishlist

### ISSUE THREAD

**Issue by LaunchedBerry07** [pop-os/shop#425]
**Feature requested:**

Wishlist or "download/install later" list

**Description:**

An user list - as the Google Play Store wishlist - that would save data locally instead of online

**Comment by leviport** [comment]
I'm not sure I understand the use case for this feature. Since the applications in Pop Shop do not cost money, why would a user not just install them straight away? 

Also, what would be the purpose of caching applications offline without installing them? If a user has network access to download the applications into a cache, why wouldn't they just want to install them right away?

**Comment by LaunchedBerry07** [comment]
@leviport 

Example: An user who wants to test out multiple apps of the same type (e.g. several Calendar apps) but doesn't want to install all of them right away for resource-saving reason. That user could then add those apps to his "install later" list.

And I'm sorry, I meant "saving data" as in "saving the app URLs" on disk, per example - not the whole packages.

Does it make a bit more sense to you?

**Comment by leviport** [comment]
Yeah I think I'm understanding better now. Thanks for the explanation and the feature request!

**Comment by LaunchedBerry07** [comment]
@leviport Well, I thank you and your team for your amazing work! Love Pop OS!
